src/utilities/createHabitData.ts

```diff
@@ -0,0 +1,36 @@
+import lightFormat from "date-fns/lightFormat";
+
+export function createHabitData() {
+  const habitData = [];
+  const rewardsDays = [
+    Math.ceil(Math.random() * 30),
+    Math.ceil(Math.random() * 30),
+    Math.ceil(Math.random() * 30),
+    Math.ceil(Math.random() * 30),
+    Math.ceil(Math.random() * 30),
+    Math.ceil(Math.random() * 30),
+  ];
```

Comment:
This may produce a case where one day may have multiple rewards - I think that it should not happen.